### PR TITLE
Improvement tweaks

### PR DESCRIPTION
**Pull request by bsmthal**
Changed the way the acceleration indicator LEDs light up so they are (1) more useful and (2) more intuitive

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -88,9 +88,23 @@ void loop() {
...
+    
+    if (y < 0)
+      Yboard.set_led_color(7, 0, map(y, 0, -1000, 0, 255), 0);
+    else
+      Yboard.set_led_color(7, map(y, 0, 1000, 0, 255), 0, map(y, 0, 1000, 0, 255));
+
+    if (z <0)
+      Yboard.set_led_color(8, 0, 0, map(z, 0, -1000, 0, 255));
+    else
+      Yboard.set_led_color(8, map(z, 0, 1000, 0, 255), map(z, 0, 1000, 0, 255), 0);
```
Comment (philipbl) [inline src/main.cpp]: Please change all of the if statements so they have braces.

**Review by philipbl — CHANGES_REQUESTED** [review]
Remember, you don't have to submit a new PR. You can just push to this branch and the PR will get updated.

File: src/main.cpp (hunk 1 above)
Comment (bsmthal) [reply src/main.cpp]: It is done.

Comment (philipbl) [reply src/main.cpp]: It looks like you still need to do it to lines 84 to 107.

Comment (bsmthal) [reply src/main.cpp]: Oh oops. I'll get on that tomorrow morning.

Comment (bsmthal) [reply src/main.cpp]: Well now it looks all ugly, but it is done.

**Review by philipbl — APPROVED** [review]

**Pull request merged**